ortho_seq_code/tests/test_orthoseqs.py

```diff
@@ -127,7 +127,7 @@ def test_nucleotide_first_order(
         nucleotide_params_first_order = nucleotide_params_first_order._replace(
             out_dir=location
         )
-        location += "[0]"  # Test file paths already exists, but location isn't updated with out_dir, this will update it
+        location += "0"  # Test file paths already exists, but location isn't updated with out_dir, this will update it
```

Comment:
Why does "0" need to be appended to all of these new temp directory names?

Reply:
this was to deal with the renaming out_dir issue... @nkoix can add a comment here to clarify as I'm forgetting at the moment..

Reply:
Our very first approach to solve the out_dir parameter from overwriting any old data was to add brackets, with a number in between it (ex. if `output` is the out_dir, and it already exists, then it would become `output[0]`). We're doing something different now, that's legacy code such that it would allow `pytest` to pass, since apparently pytest creates multiple output directories.